cli/src/main.rs

```diff
@@ -83,18 +83,22 @@ fn build(release: bool) -> Result<()> {
 
     let workspace_dir = cargo::workspace_directory()?;
     let target_dir = cargo::target_directory()?;
-    let binary_name = cargo::binary_name()?;
-    fs::create_dir_all(workspace_dir.join("workflow"))?;
+    let binary_names = cargo::binary_names()?;
+    let workflow_dir = workspace_dir.join("workflow");
+    fs::create_dir_all(&workflow_dir)?;
 
-    let src = target_dir.join(mode.dir()).join(&binary_name);
-    let dst = workspace_dir.join("workflow").join(&binary_name);
-    fs::copy(&src, &dst)?;
+    binary_names.iter().for_each(|binary_name| {
+        let src = target_dir.join(mode.dir()).join(binary_name);
+        let dst = workflow_dir.join(binary_name);
+        fs::copy(&src, &dst).unwrap();
+    });
```

Comment:
I would prefer a normal `for` loop here and then you can bubble up the io error

```suggestion
    for binary_name in &binary_names {
        let src = target_dir.join(mode.dir()).join(binary_name);
        let dst = workflow_dir.join(binary_name);
        fs::copy(&src, &dst)?;
    }
```